### ISSUE TITLE
The module does not work with Node.js

### ISSUE THREAD

**Issue by Daeren** [manticoresoftware/manticoresearch-javascript#15]
Node.js v18.9.1  
OS: Win 11  
  
Code:  
```  
const Manticoresearch = require('manticoresearch');  
const client = Manticoresearch.ApiClient();  
  
client.basePath = "http://localhost:9308";  
```  
  
Result:  
```  
client.basePath = "http://localhost:9308";  
                ^  
TypeError: Cannot set properties of undefined (setting 'basePath')  
```  
  
Something is wrong with the implementation in ApiClient.js (???):  
root.Manticoresearch.ApiClient = factory(root.superagent, root.querystring);

**Comment by githubmanticore** [comment]
➤ Nick Sergeev commented:
 

 
Fixed in 8dcbe1f9

**Comment by githubmanticore** [comment]
➤ Nick Sergeev commented:
 

 
Fixed in https://github.com/manticoresoftware/manticoresearch-javascript/commit/8d4f9af8c0ed684a130e8ff07b79dd6eadb2171c 
 
There was an error in the documentation: `new` operator was missed in the `ApiClient` instance creation command.
